utils/utils.go

```diff
@@ -38,25 +39,38 @@ import (
 type ContainerRuntime string
 
 const (
-	DOCKER ContainerRuntime = "docker"
-	PODMAN ContainerRuntime = "podman"
+	DOCKER     ContainerRuntime = "docker"
+	PODMAN     ContainerRuntime = "podman"
+	CONTAINERD ContainerRuntime = "containerd"
+
+	NERDCTL = "nerdctl"
+	MACOS   = runtime.GOOS == "darwin"
 
 	marinerImageVariantName = "mariner"
 
 	socketFormat = "%s/dapr-%s-%s.socket"
 )
 
+var IsValidRuntimeOnMacos = func(containerRuntime string) bool {
+	containerRuntime = strings.TrimSpace(containerRuntime)
```

Comment:
See this comment- https://github.com/dapr/cli/issues/1232#issuecomment-1447590195